app/src/main/java/app/revanced/integrations/videoplayer/CopyVideoUrlTimestampButton.java

```diff
@@ -0,0 +1,21 @@
+package app.revanced.integrations.videoplayer;
+
+import app.revanced.integrations.patches.CopyVideoUrlPatch;
+import app.revanced.integrations.settings.SettingsEnum;
+
+public class CopyVideoUrlTimestampButton {
+    static BottomControlButton _button;
```

Comment:
Is there a specific reason the button classes do not extend on it?

Reply:
I thought about that but couldn't figure out how it would work.

Reply:
I'll give it a look then.

Reply:
I have pushed some changes, please review & make a final test so we can merge it.

Reply:
Also pushed a change to remove video time hook, will test in a bit.

Reply:
Added null checks to button instances, since `changeVisibility` probably gets called before the button gets initialized. After doing this change, everything works as intended.